package.json

```diff
@@ -2,6 +2,7 @@
   "private": true,
   "scripts": {
     "run:server": "dotnet run --cofiguration Release --project ./GomokuServer/src/GomokuServer.Api/GomokuServer.Api.csproj",
+    "generate-hub-types:server": "dotnet tsrts --project ./GomokuServer/src/GomokuServer.Api/GomokuServer.Api.csproj --output ./GomokuClient/packages/gomoku-core/src/types/hubs  --assemblies",
```

Comment:
Please also extend Makefile to add this script

Reply:
done